server/views/pages/index.njk

```diff
@@ -36,8 +36,8 @@
   <div class="govuk-grid-column-one-quarter">
     <p class="govuk-!-font-weight-bold">Staff facing</p>
     <ul class="govuk-list govuk-list--bullet">
-    <li><a href="">v1</a>
-    <p>Last updated: TBC</p>
+    <li><a href="/staff">v1</a>
```

Comment:
These are **supervisor** facing screens, not staff facing screens 
- **staff** are placement co-ordinators and probation practitioners. We will be using this section for the PP part of the access journey